# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/stressapptest/hotfix-glibc.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- stressapptest-1.0.9/src/os.h.vanilla	2023-09-30 11:47:21.800044576 +0200
+++ stressapptest-1.0.9/src/os.h	2023-09-30 11:47:34.572045889 +0200
@@ -18,6 +18,7 @@
 
 #include <dirent.h>
 #include <unistd.h>
+#include <stdint.h>
 #include <sys/syscall.h>
 
 #include <string>
